src/api/Gemfile

```diff
@@ -75,8 +75,6 @@ gem 'influxdb-rails'
 gem 'deep_cloneable'
 # Server-side datatables
 gem 'ajax-datatables-rails'
-# Add syntax highlight in ruby
-gem 'coderay'
```

Comment:
@eduardoj I think we cannot remove coderay, this is still used in non beta and other places

Reply:
Ops, true. I restored the gem requirement. I will add code to keep using codery for rendering diffs for non-beta.

Reply:
Code add. Please take a look again.